src/utils/convertStringToSha256Buffer.js

```diff
@@ -10,6 +10,9 @@ const encodeText = str => {
 export default message => {
   const data = encodeText(message);
   const crypto = window.msCrypto || window.crypto;
+  if (!crypto.subtle) {
```

Comment:
I would log a warning here that we are not able to hash IDs on a non-secure page. Maybe we should double check that the protocol is HTTP before we log. So, inside `if (!crypto.subtle)..` add another if to check the protocol then warn the user.

Reply:
Should we pass logger to the util?
What is the value in checking for HTTP, if the browser doesn't support hashing.
Shouldn't the warning be
`Your browser doesn't support hashing...`?

Reply:
As discussed on the call, let's not worry about the protocol.